src/Bicep.LangServer/Completions/BicepCompletionContext.cs

```diff
@@ -169,6 +194,34 @@ public static BicepCompletionContext Create(IFeatureProvider featureProvider, Co
                 activeScopes);
         }
 
+        private static bool IsDisableNextLineDiagnosticsDirectiveStartContext(BicepFile bicepFile, int offset, List<SyntaxBase> matchingNodes)
+        {
+            var tokens = matchingNodes.Where(x => x is Token token &&
```

Comment:
FYI you only need to check `matchingNodes[^1]` - the nodes are ordered hierarchically, so the token will always be at the end.

Reply:
That's interesting. Thanks!

Updated here:
https://github.com/Azure/bicep/commit/93306f521326322916034af955a75e55a966031a